examples/spacy/text_classification.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# coding: utf8
+"""Train a convolutional neural network text classifier on the
+IMDB dataset, using the TextCategorizer component. The dataset will be loaded
+automatically via Thinc's built-in dataset loader. The model is added to
+spacy.pipeline, and predictions are available via `doc.cats`. For more details,
+see the documentation:
+* Training: https://spacy.io/usage/training
+
+Compatible with: spaCy v2.0.0+
+"""
+from __future__ import unicode_literals, print_function
+import plac
+import random
+from pathlib import Path
+import thinc.extra.datasets
+
+import mlflow
+import mlflow.spacy
+import spacy
+from spacy.util import minibatch, compounding
+
+
+@plac.annotations(
```

Comment:
Can we use `click` for this?

Here are a couple of references:
1. [cli.py](https://github.com/mlflow/mlflow/blob/master/mlflow/cli.py)
2. [hyperparam/train.py](https://github.com/mlflow/mlflow/blob/master/examples/hyperparam/train.py#L111)